src/nfts/types.ts

```diff
@@ -1,3 +1,5 @@
+import { NetworkId } from 'src/transactions/types'
```

Comment:
It's a bit strange that networkId lives in transaction types, when it's widely shared. Maybe we should have a src/shared/types.ts, but that's outside of the scope of this PR.